adam/experiment/observer.py

```diff
@@ -636,13 +633,20 @@ class HTMLLoggerPreObserver(  # pragma: no cover
     html_logger: LearningProgressHtmlLogger = attrib(
         init=True, validator=instance_of(LearningProgressHtmlLogger), kw_only=True
     )
-    candidate_accuracy_observer: Optional[  # type: ignore
-        CandidateAccuracyObserver
-    ] = attrib(kw_only=True, validator=optional(instance_of(CandidateAccuracyObserver)))
+
+    # fmt: off
```

Comment:
What is this control (?) comment doing?

Reply:
It prevents reformatting file, which otherwise breaks the type: ignore. You and I hada conversation about that typing bug a while back.

Reply:
Cool! I'm going to make a note about this in the future in case I need that.